lib/elixir/lib/calendar.ex

```diff
@@ -1420,4 +1505,33 @@ defmodule DateTime do
       calendar.to_string(dt)
     end
   end
+
+  @doc """
+  Compares two `DateTime` structs. Returns :gt if first datetime is later than the second
+  and :lt for vice versa. If the two datetimes are equal :eq is returned
+  Note that both utc and stc offsets will be taken into account when comparison is done.
+
+  ## Examples
+
+     iex> dt1 = %DateTime{year: 2000, month: 2, day: 29, zone_abbr: "AMT",
```

Comment:
The `iex>` should be four spaces (it is three). The line below should align `hour:` right under `year:`.